migracja.rst

```diff
@@ -0,0 +1,236 @@
+**Laboratorium EE**
+
+*30 XII 2016*
+
+Model migracji z systemów EZD PUW i eDok
+========================================
+
+Wstęp
+<<<<<
+
+Operacja migracji danych z instalacji istniejącego systemu EZD do innego system EZD, powinna być
+realizowana za pośrednictwem pliku pośredniego (lub pewnej liczby plików) — a nie on-line.
```

Comment:
Dwie podane alternatywy (plik/on-line) nie wykluczają się nawzajem.

Należałoby przeformułować to zdanie, aby nie sugerowało, że alternatywy się wykluczają.